concourse/pipelines/destroy-deployer.yml

```diff
@@ -131,6 +131,9 @@ jobs:
           - |
             . vpc-terraform-outputs/tfvars.sh
             touch concourse.crt concourse.key
+
+            export TF_VAR_git_rsa_id_pub="anything"
```

Comment:
I think this should go in the `params` hash alongside `TF_VAR_env` and `TF_VAR_system_dns_zone_name`.